tests/lib/ignored-paths.js

```diff
@@ -244,6 +244,12 @@ describe("IgnoredPaths", () => {
             assert.strictEqual(ignoredPaths.ignoreFileDir, path.dirname(ignoreFilePath));
         });
 
+        it("should set the common ancestor directory of cwd and ignorePath", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, ignorePath: ignoreFilePath, cwd: path.resolve(ignoreFilePath, "../../../testcwd") });
+
+            assert.strictEqual(ignoredPaths.getBaseDir(), path.resolve(ignoreFilePath, "../../.."));
+        });
+
```

Comment:
Is it feasible to add a test for when the baseDir is only a windows drive letter?

Reply:
The base directory was only a Windows drive letter on Appvoyer environments.
But I'm not sure if we can write such a test safely or not, because of permissions. On my PC, the base directory has often become the home directory of my login user because the temporary directory for tests is in ` ~\AppData\Local\Temp`.